app/src/actions/search.js

```diff
@@ -1,30 +1,43 @@
-import { GET_SEARCH_RESULTS } from '../actions';
+import { SET_SEARCH_RESULTS } from '../actions';
 import 'whatwg-fetch';
 
+/**
+ * Reset the search results to their initial state
+ *
+ * @export resetSearchResults
+ * @returns {Object}
+ */
+export function resetSearchResults() {
+  return {
+    type: SET_SEARCH_RESULTS,
+    payload: {
+      entries: [],
+      count: -1
+    }
+  };
+}
+
 export function getSearchResults(searchTerm) {
   return (dispatch, getState) => {
     const state = getState();
 
     if (!state.user.token) {
```

Comment:
I agree with you. I started with a return statement but eslint wouldn't let me because of inconsistent statements. Maybe we can skip the rule.